src/axolotl/utils/models.py

```diff
@@ -48,6 +48,29 @@
 LOG = logging.getLogger("axolotl")
 
 
+# copied from accelerator.FullyShardedDataParallelPlugin
+def get_module_class_from_name(module, name):
+    """
+    Gets a class from a module by its name.
+    Args:
+        module (`torch.nn.Module`): The module to get the class from.
+        name (`str`): The name of the class.
+    """
+    modules_children = list(module.children())
+    if module.__class__.__name__ == name:
+        return module.__class__
+
+    if len(modules_children) == 0:
+        return None
+
+    for child_module in modules_children:
+        module_class = get_module_class_from_name(child_module, name)
+        if module_class is not None:
+            return module_class
+
+    raise ValueError(f"module class {name} not found in {module.__class__.__name__}")
```

Comment:
```suggestion
```

I added this when copying it over, but it's probably incorrect since it recurses above and should raise if not found in a child.